### PR TITLE
Add handling of media links in customs models

### PR DESCRIPTION
**Pull request by buddhaCode**
Explain how to prevent media entities from been moved to trash bin, when used in custom models.

### CODE REVIEW

**Comment by mitelg** [comment]
hey @buddhaCode 
thanks for this good contribution :+1: 
I don't know if this is the right place in the docs. 
Maybe it deserves an own article under "[Tutorials](https://developers.shopware.com/developers-guide/tutorials/)". What do you think about that?

**Comment by uehler** [comment]
or maybe just link to https://developers.shopware.com/developers-guide/shopware-5-upgrade-guide-for-developers/#garbage-collector ?

File: source/developers-guide/models/index.md [hunk 1]
```diff
@@ -73,6 +73,42 @@ _The name of the model is always on the left side of the model, with which the a
 
 </div>
 
+## Linking to Media entities
+
+Media entities savour a special threatment in Shopware. They will be moved to the trash bin colletion by the media garbage collection, if they aren't referenced by another entity anymore. If you want to use media references in your custom model, you should annouce this to the garbage collector. Otherwise, media entities maybe moved to trash bin by mistake. You can announce references in associations and occurrences of `<img>`-tags in HTML text fields. You just need to subscribe the `Shopware_Collect_MediaPositions` event and return an `ArrayCollection` with `MediaPosition` objects, which define your reference.
```
Comment (uehler) [inline source/developers-guide/models/index.md]: `you should annouce`
announce

**Comment by mitelg** [comment]
@uehler good idea :+1:

**Comment by mitelg** [comment]
hey @buddhaCode 
what is the status here?

**Comment by mitelg** [comment]
hey @buddhaCode 
This PR was closed due to inactivity. If this change is still important to you, feel free to create a new pull request.

**Pull request closed without merging**